sdk/contentsafety/Azure.AI.ContentSafety/README.md

```diff
@@ -1,43 +1,91 @@
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing [Azure AI Content Safety][contentsafety_overview] instance.
 
-> You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/) and [Cosmos DB account](https://docs.microsoft.com/azure/cosmos-db/account-overview) (SQL API). In order to take advantage of the C# 8.0 syntax, it is recommended that you compile using the [.NET Core SDK](https://dotnet.microsoft.com/download) 3.0 or higher with a [language version](https://docs.microsoft.com/dotnet/csharp/language-reference/configure-language-version#override-a-default) of `latest`.  It is also possible to compile with the .NET Core SDK 2.1.x using a language version of `preview`.
 
 ### Authenticate the client
 
-If your library requires authentication for use, such as for Azure services, include instructions and example code needed for initializing and authenticating.
+#### Get the endpoint
+You can find the endpoint for your Azure AI Content Safety service resource using the [Azure Portal][azure_portal] or [Azure CLI][azure_cli_endpoint_lookup]:
 
-For example, include details on obtaining an account key and endpoint URI, setting environment variables for each, and initializing the client object.
+```bash
+# Get the endpoint for the Azure AI Content Safety service resource
+az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "properties.endpoint"
+```
 
-## Key concepts
+#### Get the API key
+
+The API key can be found in the [Azure Portal][azure_portal] or by running the following [Azure CLI][azure_cli_key_lookup] command:
+
+```bash
+az cognitiveservices account keys list --name "<resource-name>" --resource-group "<resource-group-name>"
+```
+
+#### Create a ContentSafetyClient with AzureKeyCredential
+
+Pass the API key as a string into an instance of `AzureKeyCredential`.
+
+```csharp
+string endpoint = "https://<my-custom-subdomain>.cognitiveservices.azure.com/";
+string key = "<api_key>";
 
-The *Key concepts* section should describe the functionality of the main classes. Point out the most important and useful classes in the package (with links to their reference pages) and explain how those classes work together. Feel free to use bulleted lists, tables, code blocks, or even diagrams for clarity.
+ContentSafetyClient client = new ContentSafetyClient(new Uri(endpoint), new AzureKeyCredential(key));
+```
+
+## Key concepts
 
-Include the *Thread safety* and *Additional concepts* sections below at the end of your *Key concepts* section. You may remove or add links depending on what your library makes use of:
+### Harm categories
+Content Safety recognizes four distinct categories of objectionable content.
+|Category	|Description  |
```

Comment:
I recommend putting space between paragraphs, tables, headings, lists (not items), etc. If you install the Microsoft Learn Pack, it has a linter that will help format this.

While the README.md does get rendered in many cases, we want it to be readable even as plain text - one of the goals of markdown.

Reply:
Thanks for the suggestion! It's very helpful. Reformat the readme by linter.